File: RepoTransBench/target_projects/Java/Go/cyfonly_FLogger/public_tests/flogger_test_public.go
package public_tests

import (
	"testing"
)

type DummyLogger struct{}

func (l *DummyLogger) Info(msg string) {}
func (l *DummyLogger) Warn(msg string) {}

func GetLogger() *DummyLogger {
	return &DummyLogger{}
}

func TestLoggerInfoAndWarnPublic(t *testing.T) {
	logger := GetLogger()
	logger.Info("Public INFO message for FLoggerTestPublic")
	logger.Warn("Public WARN message for FLoggerTestPublic")
}

File: RepoTransBench/target_projects/Java/C#/cyfonly_FLogger/public_tests/FLoggerTestPublic.cs
using Xunit;

namespace cyfonly_FLogger.Tests.Public
{
    public class FLoggerTestPublic
    {
        [Fact]
        public void TestLoggerInfoAndWarnPublic()
        {
            var logger = cyfonly_FLogger.FLogger.GetInstance();
            logger.Info("Public INFO message for FLoggerTestPublic");
            logger.Warn("Public WARN message for FLoggerTestPublic");
        }
    }
}